nanomq/pub_handler_test.c

```diff
@@ -0,0 +1,92 @@
+#include <assert.h>
+#include <stdio.h>
+
+#include "include/nanomq.h"
+#include "include/pub_handler.h"
+
+int
+main()
+{
+	int         rv      = -1;
+	reason_code rv_rc   = -1;
+	bool        rv_bool = false;
+	// init work
+	nano_work *work;
+	work            = nng_alloc(sizeof(*work));
+	work->proto_ver = MQTT_PROTOCOL_VERSION_v311;
+	nng_msg                  *msg;
+	struct pub_packet_struct *pub_packet;
+	nng_msg_alloc(&msg, 0);
+	pub_packet                         = nng_zalloc(sizeof(*pub_packet));
+	work->msg                          = msg;
+	work->pub_packet                   = pub_packet;
+	int                  remaining_len = 18;
+	struct fixed_header *fix_hd        = nng_alloc(sizeof(*fix_hd));
+	fix_hd->qos                        = 0;
+	fix_hd->packet_type                = PUBLISH;
+	int topic_len                      = 7;
+	int data_len                       = 4;
```

Comment:
Did you forget to format it? Or make it clear

Reply:
I will format it.